t04_refactored.py

```diff
@@ -170,8 +174,52 @@ def team_12_adv():
 
 
 def team_13_adv():
-    pass
-    # TODO Add your code here
+    """
+    Andy Hill
+    Nyan Lin Zaw
+    https://docs.google.com/document/d/1-lO7SPynvH8GxfrMS5xBYxhiu9YZs-9oWcI3YCdIKTA/edit
+    :return: none
+    """
+
+    from time import sleep
```

Comment:
Imports don't belong inside functions. This import is already done, at the beginning of this file. Please remove.